### PR TITLE
Fix Golorn Frostbeard and Curator Thorius

### PR DESCRIPTION
**Pull request by banehammer**
https://github.com/azerothcore/azerothcore-wotlk/issues/7766

Thank you @Rushor ;) (y)

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1632773438019686726.sql [hunk 1]
```diff
@@ -0,0 +1,65 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1632773438019686726');
+
+
+DELETE FROM `waypoint_data` WHERE `id`=2190;
+(2190, 1, -5640.79, -501.86, 396.715, 0, 30000, 0, 0, 100, 0),
+(2190, 2, -5640.07, -497.906, 396.669, 0, 0, 0, 0, 100, 0),
+(2190, 3, -5635.95, -495.509, 396.678, 0, 0, 0, 0, 100, 0),
+(2190, 4, -5632.68, -496.318, 396.702, 0, 30000, 0, 0, 100, 0),
+(2190, 5, -5635.95, -495.509, 396.678, 0, 0, 0, 0, 100, 0),
+(2190, 6, -5640.07, -497.906, 396.669, 0, 0, 0, 0, 100, 0);
```
Comment (syssneck) [inline data/sql/updates/pending_db_world/rev_1632773438019686726.sql:10]: ```INSERT INTO `waypoint_data` (...) Values``` missing here

Comment (banehammer) [reply data/sql/updates/pending_db_world/rev_1632773438019686726.sql:10]: Will edit when back home

File: data/sql/updates/pending_db_world/rev_1632773438019686726.sql [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+(18870, 36, -4683.05, -1314.02, 503.382, 0, 0, 0, 0, 100, 0),
+(18870, 37, -4675, -1312.6, 503.382, 0, 0, 0, 0, 100, 0),
+(18870, 38, -4664.88, -1306.13, 503.382, 0, 0, 0, 0, 100, 0),
+(18870, 39, -4662.04, -1298.12, 503.382, 0, 0, 0, 0, 100, 0),
+(18870, 40, -4659.43, -1288.83, 503.382, 0, 0, 0, 0, 100, 0),
+(18870, 41, -4654.31, -1281.54, 503.382, 0.959392, 75000, 0, 0, 100, 0);
+
+
+DELETE FROM `waypoint_data` WHERE `id`=188700;
+INSERT INTO `waypoint_scripts` (`id`, `delay`, `command`, `datalong`, `datalong2`, `dataint`, `x`, `y`, `z`, `o`, `guid`) VALUES
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1632773438019686726.sql:60]: always delete before insert.

**Review by Kitzunu — CHANGES_REQUESTED** [review]

**Comment by Kitzunu** [comment]
Please do not remove the PR template. Edit and fill it in please.

**Comment by banehammer** [comment]
Resolved the errors.

**Comment by Asthalor** [comment]
Tested

Curator Thorius seems to have a good route. I dont know the blizzlike one, but he uses emotes and seems like a normal route. All good.

image

Golorn Frostbeard seems like he is very shy, because it is not facing us and is moving between the 2 boxes behind him. 

image

image

Without visual patches(if its important)

image

Wowhead image is looking at us. Maybe he should be waiting for some time looking at us in between routes?

image

**Comment by banehammer** [comment]
Hmmm Golorn that weirdo. He is supposed to stop being shy with this commit. Wait...

**Comment by banehammer** [comment]
Okay updated the orientation.

**Comment by Asthalor** [comment]
Ok. Now is looking at us.

image

Seems good to me.

**Pull request closed without merging**